include/seastar/http/request.hh

```diff
@@ -87,6 +87,7 @@ struct request {
     std::unordered_map<sstring, sstring> chunk_extensions;
     sstring protocol_name = "http";
     noncopyable_function<future<>(output_stream<char>&&)> body_writer; // for client
+    int listener_idx;
```

Comment:
@mmaslankaprv , there's a PR #1851 that adds remote and (!) local address to http::request. Would the knowledge of local address be a good replacement for the listener_idx thing?

Reply:
yes, it would be even better, i will defer the update till #1851 is merged, thank you for this suggestion

Reply:
@mmaslankaprv #1851 is merged (FYI)